packages/esm-amd-loader/src/esm-amd-loader.ts

```diff
@@ -545,7 +545,7 @@ function resolveUrl(urlBase: NormalizedUrl, url: string): NormalizedUrl {
     // Already a fully qualified URL.
     return url as NormalizedUrl;
   }
-  return normalizeUrl(urlBase + url);
+  return normalizeUrl(url[0] === '/' ? url : urlBase + url);
```

Comment:
We should also document on normalizeUrl that it can take urls like `/foo/bar` as input

Reply:
Done.